toolbox/pkg_check/main_test.go

```diff
@@ -149,10 +149,21 @@ func TestFailedCheck(t *testing.T) {
 }
 
 func TestMain(m *testing.M) {
+	if _, err := os.Stat(tmpDir); os.IsNotExist(err) {
```

Comment:
You might want to use https://golang.org/pkg/io/ioutil/#TempFile to make your test more predictable. As of now if I run your test concurrently they will fail. You want to make sure that each file name is unique.